regression/cbmc/pointer-to-string-function-parameters/test.c

```diff
@@ -0,0 +1,119 @@
+#include <assert.h>
+#include <stddef.h>
+
+int is_printable_char(char c)
```

Comment:
Rather than duplicate this function between tests, stick it in a common include file? (I think a couple of other regression tests do something similar to share files between tests)

Reply:
I've seen that pattern, I personally don't like it very much. My opinion on that is that test should be mostly self contained, and if they're so complicated you feel the need to factor parts out you'll probably want to make them easier. In this case, I think just shortening the function with conditions like you suggested would be better.